client/lib/webauthn/index.js

```diff
@@ -0,0 +1,141 @@
+import wpcom from 'lib/wp';
+var _backend;
+
+const POST = 'POST';
+
+function strToBin(str) {
+	str = str.replace(/[-_]/g, function(m) { return m[0] === '-' ? '+' : '/'; });
```

Comment:
This looks a bit fishy, we're changing the string before converting it to binary but below in `binToStr()` we're not reversing the change. Does this conversion happen in PHP? Is this just broken?

Reply:
The values that come from the API side are encoded using with the base64url, which `atob` doesn't handle. That's why it's converted to the standard `base64` encoding.
We can potentially add the same conversion to binToStr to avoid this confusion -- the PHP version handles the `base64`, and `base64url` encodings.